.github/run_browserstack_acceptance.sh

```diff
@@ -5,13 +5,15 @@ GITHUB_BRANCH=${GITHUB_REF#refs/heads/}
 export BROWSERSTACK_BUILD_ID="${GITHUB_BRANCH} - ${GITHUB_RUN_ID}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
+export BROWSER=$1
+export BROWSERSTACK_DEBUG="true"
+export BROWSERSTACK_CONSOLE="verbose"
+export BROWSERSTACK_NETWORK_LOGS="true"
+export BROWSERSTACK_DISPLAY_RESOLUTION="1920x1080"
 
-if [[ $GITHUB_BRANCH == release/*
-  || $GITHUB_BRANCH == hotfix/*
-  || $GITHUB_BRANCH == master
-  || $GITHUB_BRANCH == support/* ]]
+if [[ $BROWSER == 'browserstack:ie@11.0' ]]
```

Comment:
maybe we can just have this script take in a concurrency arg, and have the different jobs in the acceptance test github action pass in a number. What do you think?

Reply:
sure, updated